src/components/UpdatesFeed/index.test.js

```diff
@@ -0,0 +1,80 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+import useRecentActivity from "queries/useRecentActivity";
+
+import UpdatesFeed from "../UpdatesFeed";
+
+import { HOME_TAB_KEYS } from "constants/home";
+
+jest.mock("queries/useRecentActivity");
+
+describe("UpdatesFeed tests", () => {
+	let mockData, setSelectedTab;
```

Comment:
Please declare these as `const` above the `describe` call

Reply:
We'll need to have mock data for each of the query calls, to be used in tests. At some point, we may want to create a `src/mocks` directory and move these mock data into `JSON` files there. We can then `import` them as needed into the test files. Something to keep in mind as we add more test files and start to copypasta the mocks.

Reply:
updated in e791791, for now left them in the test file itself.